layout.js

```diff
@@ -27,111 +64,164 @@ module.exports = function (images, opts) {
     });
   }
 
-  // handle large images
-  images = images.filter(function (image) {
-    if (image.contentWidth <= maxSize && image.contentHeight <= maxSize) {
-      return true;
+  var finalpass = 4;
```

Comment:
Curious about the significance of `4` here; is that just balancing the time it takes to sprite vs. diminishing returns or should it always be `4`?